core/schema/sdk.go

```diff
@@ -125,7 +127,7 @@ func parseSDKName(sdkName string) (SDK, string, error) {
 	}
 
 	// inbuilt sdk go/python/typescript currently does not support selecting a specific version
-	if slices.Contains([]SDK{SDKGo, SDKPython, SDKTypescript}, SDK(sdkNameParsed)) && hasVersion {
+	if slices.Contains([]SDK{SDKGo, SDKPython, SDKTypescript, SDKJava}, SDK(sdkNameParsed)) && hasVersion {
```

Comment:
Needs to be moved to the condition below, next to php and elixir.

Reply:
👍 I missed that when I extracted it from the built in list